jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/DeleteObjectAction.kt

```diff
@@ -13,22 +13,19 @@ import software.amazon.awssdk.services.s3.S3Client
 import software.amazon.awssdk.services.s3.model.Delete
 import software.amazon.awssdk.services.s3.model.DeleteObjectsRequest
 import software.amazon.awssdk.services.s3.model.ObjectIdentifier
-import software.aws.toolkits.core.telemetry.TelemetryNamespace
 import software.aws.toolkits.jetbrains.core.AwsClientManager
 import software.aws.toolkits.jetbrains.services.s3.S3VirtualBucket
 import software.aws.toolkits.jetbrains.services.s3.S3VirtualDirectory
 import software.aws.toolkits.jetbrains.services.s3.bucketEditor.S3KeyNode
 import software.aws.toolkits.jetbrains.services.s3.bucketEditor.S3TreeTable
-import software.aws.toolkits.jetbrains.services.telemetry.TelemetryService
 import software.aws.toolkits.jetbrains.utils.notifyInfo
 import software.aws.toolkits.resources.message
 import javax.swing.tree.DefaultMutableTreeNode
 
 class DeleteObjectAction(
     private var treeTable: S3TreeTable,
     val bucket: S3VirtualBucket
-
-) : AnActionButton("Delete Object", null, AllIcons.Actions.Cancel), TelemetryNamespace {
+) : AnActionButton(message("s3.delete.object.action"), null, AllIcons.Actions.Cancel) {
```

Comment:
Replace this with `AnActionButtonWrapper`?

Reply:
Sorry. Missed it by mistake.